src/signals/incident-management/containers/Dashboard/components/BarChart/BarChart.test.tsx

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { render, screen } from '@testing-library/react'
+
+import { BarChart } from './BarChart'
+
+describe('BarChart', () => {
+  it('should render correctly', () => {
+    render(<BarChart />)
+
+    const title = screen.queryByRole('heading', {
```

Comment:
You should use getByRole here. You would use queryBy when you expect something not to be there. QueryBy will then return null, getBy will return an error: 
https://testing-library.com/docs/queries/about/

Reply:
Personally I like it better when you get an null back instead of an error. An error could be caused by anything, a null gives more information. But I have no skin in the game, so I changed it for you.